libs/libcommon/src/libcommon/simple_cache.py

```diff
@@ -121,6 +153,13 @@ class CachedResponse(Document):
 CachedResponse.split.required = False  # type: ignore
 
 
+def validate_content_size(content: Any, content_max_size: int) -> None:
```

Comment:
I was thinking of calling it inside "upsert_response", as a validation step.

Reply:
As per last comment, I kept it on the worker